libs/domains/application/src/lib/slices/applications.slice.spec.ts

```diff
@@ -7,6 +7,10 @@ describe('applications reducer', () => {
       error: null,
       joinEnvApplication: {},
       statusLoadingStatus: 'not loaded',
+      defaultApplicationAdvancedSettings: {
```

Comment:
Why do you use default in the name? Could be better if you have only `applicationAdvancedSettings` no?

Reply:
The props accept boths defaultApplicationAdvancedSettings and applicationAdvancedSettings. The default one are the one set by default and this props helps me to make the difference with the actual one set on the application